src/components/Nav/Nav.js

```diff
@@ -1,5 +1,31 @@
+import { useState, useEffect } from 'react';
+import NavBar from '../Nav/NavBar/NavBar';
+import NavCategory from './ListModal/NavCategory/NavCategory';
+
 const Nav = () => {
-  return <div>hi</div>;
+  const [isListModalVisible, setIsListModalVisible] = useState(false);
+  const [token, setToken] = useState(null);
+
+  useEffect(() => {
+    if (localStorage.getItem('token')) {
+      setToken(localStorage.getItem('token'));
+    }
+  }, []);
```

Comment:
```suggestion
  useEffect(() => {
    const token = localStorage.getItem('token');
    token && setToken(token);
  }, []);
```
이렇게하면 훨씬 가독성이 좋아지겠죠?

Reply:
감사합니다 리뷰 반영했습니다 도현님.... 🙆🏽‍♂️